bundles/org.eclipse.orion.client.ui/web/plugins/pageLinksPlugin.js

```diff
@@ -43,15 +43,15 @@ define([
 			order: 10
 		});
 		
-		if (!util.isElectron) {
+		//if (!util.isElectron) {
```

Comment:
separate patch. We need to fix the electron build, because the shell requires native libraries. Similar to nodegit.